domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -85,11 +85,11 @@
     System.setProperty(LoggerFactory.LOGGER_DELEGATE_FACTORY_CLASS_NAME, "io.vertx.core.logging.Log4j2LogDelegateFactory");
   }
 
-  @BeforeClass
+  /*@BeforeClass
   public static void doesNotCompleteOnWindows() {
     final String os = System.getProperty("os.name").toLowerCase();
     org.junit.Assume.assumeFalse(os.contains("win")); // RMB-261
-  }
+  }*/
```

Comment:
If it runs on Windows please remove this code (it is still in git history).

Reply:
I managed to tun it with postgres-conf, but it still causes timeout with embedded postgres. So I uncommented the code.

Reply:
Thanks.